### DISCUSSION TITLE
Why is no endpoint alarm triggered，I configured the endpoint to alert when it reports an error. No alarm displayed

### DISCUSSION THREAD

**Discussion by ZKANGGIT** [apache/skywalking#12291] [Q&A]
<img width="680" alt="企业微信截图_20240604122924" src="https://github.com/apache/skywalking/assets/36831988/518a82e9-3b12-49f8-9e5d-f511c163fdbd">
<img width="336" alt="企业微信截图_20240604122854" src="https://github.com/apache/skywalking/assets/36831988/43064ab1-74bb-4610-905b-e1eed3321c71">
<img width="624" alt="企业微信截图_20240604122915" src="https://github.com/apache/skywalking/assets/36831988/98b28a38-a7b6-4fab-b7ae-f23f990b4636">

**Comment by wu-sheng** [comment]
Which version are you running? The alerting rule has been changed from the last several versions. And even for the old version, `endpoint_percent` is not a valid metric name. Nothing should be triggered.

**Reply by ZKANGGIT** [reply]
I'm using version 9.3.0. endpoint_percent I copied this rule from config-examples/alarm-settings.yml. I just changed some of the values

**Reply by ZKANGGIT** [reply]
How to configure an alarm when an ERROR occurs on the endpoint

**Reply by wu-sheng** [reply]
9.3 is too old, I can't remember. But at least for now, there is no such metric. Make sure it matches the name in the OAL scripts.

**Reply by ZKANGGIT** [reply]
Okay, thanks

**Reply by wu-sheng** [reply]
From metrics analyzed from traces, these metrics are available on the dashboard, and alert rules.

https://github.com/apache/skywalking/blob/7ad04062b20ecf2bdb0afc4ba8c6ce353e5ae4d8/oap-server/server-starter/src/main/resources/oal/core.oal#L53-L58

**Comment by wu-sheng** [comment]
It is better you could run OAP 10.1.0, rather than years ago releases.

**Reply by ZKANGGIT** [reply]
Which version of the jdk is used，I used 1.8.0 to report errors
![Uploading 企业微信截图_1716287481167.png…]()

**Reply by wu-sheng** [reply]
The doc is clear about this. Why asking this suddenly?

**Reply by ZKANGGIT** [reply]
Sorry, I'll just go and read the documentation
